pkgs/development/tools/cljfmt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildGraalvmNativeImage, fetchurl, nix-update-script }:
+
+buildGraalvmNativeImage rec {
+  pname = "cljfmt";
+  version = "0.11.1";
+
+  src = fetchurl {
+    url = "https://github.com/weavejester/${pname}/releases/download/${version}/${pname}-${version}-standalone.jar";
+    sha256 = "sha256-Rf/zHueOrN7Wf+POMXiebHvnJI8Ts6yKm7Rg+xoOa/k=";
+  };
+
+  extraNativeImageBuildArgs = [
+    "-H:+ReportExceptionStackTraces"
+    "-H:Log=registerResource:"
+    "--initialize-at-build-time"
+    "--diagnostics-mode"
+    "--report-unsupported-elements-at-runtime"
+    "--no-fallback"
+  ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
Would not hurt to add a simple [package test](https://nixos.org/manual/nixpkgs/stable/#sec-package-tests)?

```nix
  passthru.tests.version = testers.testVersion { package = cljfmt; };
```